torch/_subclasses/meta_utils.py

```diff
@@ -280,13 +296,16 @@ def empty_create(inner_t, inner_src, symbolic_context=symbolic_context):
                 if t.is_sparse:
                     is_leaf = safe_is_leaf(t)
                     r = callback(
-                        lambda: torch.ops.aten._sparse_coo_tensor_with_dims(
+                        lambda: torch.ops.aten._sparse_coo_tensor_with_dims_and_tensors(
                             t.sparse_dim(),
                             t.dense_dim(),
                             t.shape,
+                            t._indices().to(device="meta"),
+                            t._values().to(device="meta"),
```

Comment:
This conversion to meta looks suspicious

Reply:
You should look at the NT fakeification code, my expectation is you have to recursively fakeify indices/values

also cc @jbschlosser